### ISSUE TITLE
Experienced Developer and Architect (Ruby Python or Haskell)

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#8181]
Experienced Developer and Architect (Ruby, Python or Haskell)<br>
https://ift.tt/2LQWRyD<br>
<p><img src="https://we-work-remotely.imgix.net/logos/0006/7068/logo.gif?ixlib=rails-2.1.3&amp;w=190&amp;min-h=150" alt="Logo.gif?ixlib=rails 2.1"></p>
<p><strong>Headquarters:</strong> Berlin and Stuttgart, Germany<br>
<strong>URL:</strong> <a href="https://virtualq.io/en/career/">https://virtualq.io/en/career/</a></p>
<div>
<span>At virtualQ you get the chance to work on an innovative and&nbsp;</span><span>award winning&nbsp;</span><span>product with&nbsp;</span><span>top-notch technology and software management tools</span><span>. We have a German-American background and like the concept of&nbsp;</span><span>simplicity, design and flat hierarchy</span><span>. You have the chance to work in an awesome team atmosphere with lots for individual liberties. Do you want to change the way in which people wait on hold in customer service telephone lines with us? Are you&nbsp;</span><span>interested in topics like Machine Learning and Voice Automation</span><span>? Then you should take your chance and join us as a Senior Software Engineer and Architect.</span>
</div>
<div><span><br></span></div>
<div><span>IN A NUTSHELL &ndash; WHO WE ARE AND WHAT WE DO</span></div>
<div><br></div>
<div>virtualQ provides a&nbsp;<span>novel cloud-solution</span>&nbsp;that waits on hold during longer waiting times for callers of Customer Service Centers. Callers can spend their time productively until virtualQ alerts them to call back when it is their turn. Moreover, the&nbsp;<span>smart voice assistant</span>&nbsp;of virtualQ provides answers to frequently asked questions. Thereby, the virtualQ&nbsp;<span>enterprise software</span>&nbsp;creates an unique customer experience, which is currently turning the service industry upside down. On the company&acute;s side the&nbsp;<span>smart virtualQ algorithms</span>&nbsp;enable Customer Service Centers to manage their incoming calls automatically and more efficiently. Well-known companies like&nbsp;<span>Lufthansa Albatros</span>, German insurers&nbsp;<span>HanseMerkur and Gothar</span>&nbsp;and the Austrian railway provider&nbsp;<span>&Ouml;sterreichische Bundesbahn</span>are already using our software successfully. Several national and international institutions have recognised the innovative cloud products of virtualQ with awards and the company is backed by an&nbsp;<span>institutional investment fund</span>.</div>
<div><br></div>
<div><span>WE HAVE A LOT TO OFFER</span></div>
<div><br></div>
<ul>
<li>
<span>Results Only Work Environment</span><span>&nbsp;&ndash; we do not care when and where you work, we care about your work results. Therefore, we do not count working hours or holidays</span>
</li>
<li>
<span>Work on a</span><span>&nbsp;top-notch, novel product using the latest AI and Machine Learning technologies &ndash;&nbsp;</span><span>This is your unique chance to become a pioneer in the field of artificial intelligence and to&nbsp;</span><span>shape a new, evolving discipline</span>
</li>
<li>
<span>We love tech!&nbsp;</span><span>Thus, we work with the&nbsp;</span><span>latest technologies</span><span>&nbsp;and offer you a&nbsp;</span><span>free choice of hardware</span>
</li>
<li>
<span>Of course we are using modern&nbsp;</span><span>backend-end tech (Ruby, Rails, Elixir, Postgres)</span><span>&nbsp;and&nbsp;</span><span>front-end tech (React, Webpack, Sass, Preact...)</span>
</li>
<li>
<span>An awesome, international, open minded team &ndash; You will work with and learn from&nbsp;</span><span>highly-skilled colleagues, who are passionate about what they do&nbsp;</span><span>and who</span><span>&nbsp;value clean, well-written code</span>
</li>
<li>
<span>Flexibility at its best</span><span>&nbsp;&ndash; Join us in our&nbsp;</span><span>offices in the center of Berlin and Stuttgart</span><span>&nbsp;(Germany) or&nbsp;</span><span>work remotely from home</span>
</li>
<li><span>Workshops? Seminars? Conferences? BarCamps? Tell us what you need to develop your skills and we will make it happen!</span></li>
<li>
<span>Plenty of freedom and opportunities to grow&nbsp;</span><span>in the fastest growing service-tech-company Germany has to offer</span>
</li>
<li>
<span>Massages, chillout area, free drinks, rooftop bar, PlayStation, ping-pong table &ndash;&nbsp;</span><span>you will love our modern, well located offices!</span>
</li>
<li>
<span>Team Events, Happy Hour Thursdays and PlayStation Tournament&nbsp;</span><span>&ndash; activities are part of our daily office life</span>
</li>
<li>
<span>You prefer working remotely?</span><span>&nbsp;No problem for us! Working with online communication tools is part of our DNA, allowing us to&nbsp;</span><span>work in the same virtual office</span><span>. Moreover, our&nbsp;</span><span>quarterly team days</span><span>&nbsp;have become a highlight for all of us &ndash; no matter where we are based</span>
</li>
<li>
<span>Relocating&nbsp;</span><span>to Germany?&nbsp;</span><span>We are happy to assist you!</span>&nbsp;</li>
</ul>
<div><span><br></span></div>
<div><span>WITH THESE ACTIVITIES YOU WILL HAVE A GREAT IMPACT ON OUR BUSINESS</span></div>
<div><br></div>
<ul>
<li>
<span>Work closely with our CTO</span>&nbsp;and shape our product</li>
<li>Help build the development process of our growing team</li>
<li>
<span>Design Software and Architecture for our Stack</span>, using the latest technologies</li>
<li>Design&nbsp;<span>Voice and Audio Applications / Machine Learning Algorithms&nbsp;</span>and<span>&nbsp;Interfaces</span>, that will shape the future of virtualQ</li>
<li>
<span>Work with a team</span>&nbsp;of great Front and Backend developers</li>
<li>Support&nbsp;<span>DevOps of Amazon AWS hosted Servers, Heroku</span>&nbsp;etc.</li>
<li>Develop&nbsp;<span>Monitoring, Uptime and Security Strategies</span>
</li>
<li><span>Bring in your own ideas and creativity</span></li>
</ul>
<div><span><br></span></div>
<div><span>WITH THESE SKILLS YOU WILL BE OUR PERFECT CANDIDATE</span></div>
<div><br></div>
<div>
<span>MUST HAVES</span><br>
</div>
<ul>
<li>You know&nbsp;<span>Ruby or Python</span>&nbsp;by heart</li>
<li>You &#10084;&#65039; functional languages like&nbsp;<span>Elixir,&nbsp;</span><span>Clojure, Haskell</span>
</li>
<li>You are familiar with<span>&nbsp;AWS, Heroku</span>&nbsp;and the likes</li>
<li>You speak German OR English fluently (native or 2nd language)</li>
<li>You have got a&nbsp;<span>technical education / background</span>
</li>
<li>You do not take things for granted, instead you question the status quo</li>
</ul>
<div><span><br></span></div>
<div>
<span>NICE TO HAVE</span><br>
</div>
<ul>
<li>You have worked with some of the following systems / languages:</li>
<li>Twilio / Nexmo</li>
<li>OpsWorks</li>
<li>Chef or Puppet</li>
<li>Containers / Docker</li>
<li>Linux Servers / Web Servers</li>
<li>Open Source Databases</li>
<li>Machine Learning</li>
<li>Algorithms</li>
<li>You have worked in the Telecommunication market before</li>
</ul>
<div><span><br></span></div>
<div><span>YOUR DREAMJOB MIGHT BE JUST A CLICK AWAY. SO APPLY NOW!</span></div>
<div>Your contact person:&nbsp;<span>Anika</span>
</div>
<div><span><br></span></div>
<div>Find out more about virtualQ as an employer:&nbsp;<a href="https://virtualq.io/en/career/">https://virtualq.io/en/career/</a>
</div>
<div><br></div>
<div>We are an equal opportunities employer and welcome applications from all suitably qualified persons regardless of their race, sex, disability, religion/belief, sexual orientation or age.</div>
<div><br></div>
<p><strong>To apply:</strong> Please apply online via <a href="https://short.sg/a/1995680">https://short.sg/a/1995680</a></p>
<br>
<br>
via Remote Jobr Programming Jobs https://ift.tt/1WipnMk<br>
September 03, 2018 at 05:05AM
